### PR TITLE
Update Experimental Features

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article for review:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/index?branch=pr-en-us-3077

AB#49138495

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 1]
```diff
@@ -244,34 +238,46 @@ See also:
...
 Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is present in Microsoft Edge Canary 124.
+* This checkbox is present in Microsoft Edge Stable 122.
 
 
 <!-- ====================================================================== -->
-## Source order viewer
+## In heap snapshots, treat backing store size as part of the containing object
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:246]: review - @captainbrosset

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 2]
```diff
@@ -244,34 +238,46 @@ See also:
...
+* This checkbox is present in Microsoft Edge Canary 124.
+* This checkbox is not present in Microsoft Edge Stable 122.
 
 
 <!-- ====================================================================== -->
 ## Timeline: event initiators
 <!-- checkbox has a (?) link to this anchor wording -->
 
-Whether to include event initiators in the Timeline.
+Whether to include event initiators in the Timeline.<!-- todo: in the **Performance** tool and **Memory** tool. -->
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:269]: Perf tool, Mem tool, or both?  @captainbrosset

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:268]:
```
## Timeline: event initiators

Whether to include event initiators in the **Performance** tool.  This is useful to link a selected event to the other event that initiated it.

See also:
* [Introduction to the Performance tool](../evaluate-performance/index.md)
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 3]
```diff
@@ -440,20 +460,48 @@ Status:
...
+See also:
+* []() -->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 124.
+* This checkbox is present in Microsoft Edge Stable 122.
+
+
+<!-- ====================================================================== -->
+## Deprecate CSS \<length\> authoring tool in the Styles tab
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md]: removed entry

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 4]
```diff
@@ -149,53 +146,42 @@ The experiments checkboxes that appear in the latest version of the Canary previ
...
-<!-- ====================================================================== -->
-## Automatically pretty print minified sources
-<!-- no (?) link -->
-
-<!-- as of 115, not 114-, checkbox label is now:
-Automatically pretty print minified sources
-instead of:
-Automatically pretty print in the Sources Panel
--->
+To capture JavaScript stack traces when DOM nodes are added to the DOM at runtime, enable this experiment.  The captured stack traces are displayed in the **Stack Trace** pane of the **Elements** tool.<!-- todo: what tool contains this Stack Trace pane?  Sources tool?  Crash Analyzer tool?  Console?  where in that tool?  Update See Also accordingly -->
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:163]: What tool contains this Stack Trace pane? Sources tool? Crash Analyzer tool? Console? where in that tool?
@captainbrosset 
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/?branch=pr-en-us-3077#capture-node-creation-stacks

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:163]: The **Stack Trace** pane is in the **Elements** tool, in the sidebar, next to **Styles**, **Computed**, etc.

image

After you enable the experiment, here's how to test:

1. Open the Elements tool
2. Use a webpage that creates DOM nodes dynamically with JS, such as the demo-to-do app
3. Create a new TODO item
4. Select the new TODO item with the inspect tool
5. Open the **Stack Trace** pane.

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 5]
```diff
@@ -244,34 +236,44 @@ See also:
...
+See also:
+* [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 124.
+* This checkbox is not present in Microsoft Edge Stable 122.
+
+The present section is retained because older browsers have a (?) link pointing to it.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:276]: deleted section

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 6]
```diff
@@ -300,25 +302,29 @@ Temporarily disables the indentation markers in the **Sources** tool. The indent
...
+See also:
+* [Improved WebAssembly debugging](../whats-new/2019/12/devtools.md#improved-webassembly-debugging) in _What's New in DevTools (Microsoft Edge 80)_.
+
 Status:
-* This checkbox is not present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 124.
+* This checkbox is not present in Microsoft Edge Stable 122.
+
+The present section is retained because older browsers have a (?) link pointing to it.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:327]: deleted section

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 7]
```diff
@@ -440,20 +465,28 @@ Status:
...
+See also:
+* [More precise changes in the Changes tab](https://developer.chrome.com/blog/new-in-devtools-98/#changes)
 
 Status:
-* This checkbox is not present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 124.
+* This checkbox is not present in Microsoft Edge Stable 122.
+
+The present section is retained because older browsers have a (?) link pointing to it.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:489]: deleted section

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 8]
```diff
@@ -487,9 +528,15 @@ Status:
...
+* [Override network response headers](https://developer.chrome.com/blog/new-in-devtools-113/#network)
+* [Add columns for response headers](../network/reference.md#add-columns-for-response-headers) in _Network features reference_.
+
 Status:
-* This checkbox is not present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 124.
+* This checkbox is not present in Microsoft Edge Stable 122.
+
+The present section is retained because older browsers have a (?) link pointing to it.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:539]: deleted section

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 9]
```diff
@@ -506,8 +553,10 @@ See also:
 * [Change colors with the Color Picker](../css/reference.md#change-colors-with-the-color-picker) in _CSS features reference_
 
 Status:
-* This checkbox is not present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 124.
+* This checkbox is not present in Microsoft Edge Stable 122.
+
+The present section will be removed, because current browsers no longer have this checkbox, and older browsers don't have a (?) link pointing to this section.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:559]: deleted section

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/performance-reference.md [hunk 10]
```diff
@@ -23,13 +23,17 @@ ms.date: 05/04/2021
...
+In the **Performance** tool, the timeline events mode displays all events that are triggered while making a recording.  Each timeline event type is described below:
+* [Loading events](#loading-events)
+* [Scripting events](#scripting-events)
+* [Rendering events](#rendering-events)
+* [Painting events](#painting-events)
 
 
 <!-- ====================================================================== -->
-## Common timeline event properties
+## Properties common to all timeline events
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/performance-reference.md:34]: deleted 'timeline'

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/performance-reference.md [hunk 11]
```diff
@@ -23,13 +23,17 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # Timeline event reference
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/performance-reference.md:24]: globally updated title

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 12]
```diff
@@ -641,52 +611,87 @@ See also:
...
+
+
+<!-- ====================================================================== -->
+## Enable save and load trace with annotations in Performance Panel
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 125_
+
+Enables save and load trace with annotations in the **Performance** tool.
+<!-- todo: need desc -->
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:687]: Enable save and load trace with annotations in Performance Panel - need real description, if possible.
@captainbrosset

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 13]
```diff
@@ -902,12 +912,54 @@ This is a programmatic way to record JavaScript execution, which can be useful i
...
+
+
+<!-- ====================================================================== -->
+## Timeline: Enable track confirmation feature that can reorder or hide a track in the flame chart
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 125_
 
+Enables the track confirmation feature, which can reorder or hide a track in the flame chart.
+<!-- todo: need desc
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:930]: Timeline: Enable track confirmation feature that can reorder or hide a track in the flame chart
Need real description, if possible.  What tool is this about?
@captainbrosset

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 14]
```diff
@@ -902,12 +912,54 @@ This is a programmatic way to record JavaScript execution, which can be useful i
...
+
+
+<!-- ====================================================================== -->
+## Timeline: Enable User Timings based extensions
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 125_
+
+Enables extensions that are based on user timings.
+<!-- todo: need desc
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:951]: Timeline: Enable User Timings based extensions
Need real description, if possible.  What tool is this about?
@captainbrosset

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 15]
```diff
@@ -641,52 +610,88 @@ See also:
...
 <!-- ====================================================================== -->
-## Enable Autofill view
+## Timeline: Show postMessage dispatch and handling flows
 <!-- no (?) link -->
 
-_new as of Microsoft Edge 121_
+_new as of Microsoft Edge 124_
 
-Adds an **Autofill** tool to DevTools, to record various events that the browser triggers when a user is filling in a form by using the browser's autofill feature.  For example, when the browser remembers your address and autofills it in an address field.
+Distinguishes postMessage events from other timeline events, to easily investigate performance issues related to post messages across various threads in applications.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:656]:
```suggestion
Distinguishes events that are triggered by the `postMessage` method from other events displayed in the **Performance** tool, to easily investigate performance issues that are related to postin messages across various threads of an application.
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 16]
```diff
@@ -641,52 +610,88 @@ See also:
...
 
-![The Autofill tool](./index-images/autofill-tool.png)
+This experiment improves the **Performance** tool's timeline to help you quickly identify `postMessage` events and handlers, by distinguishing `postMessage` events from other timeline events.  Without this experiment, `postMessage` dispatch events and handler events both appear as generic scripting function call events.  With this experiment enabled:
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:658]:
```suggestion
This experiment improves the **Performance** tool's **Main** section to help you quickly identify `postMessage` events and handlers, by distinguishing `postMessage` events from other events.  Without this experiment, events triggered by dispatching and handling messages between threads of an application appear as generic scripting function call events.  With this experiment enabled:
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 12 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:687]: I couldn't find anything about this. I will ask Rob to help if he can. The bug that introduced this in Chromium was done by the Chrome team, and is unfortunately not publicly visible.

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 17]
```diff
@@ -714,24 +722,30 @@ Status:
...
+* This checkbox is present in Microsoft Edge Canary 125.
+* This checkbox is present in Microsoft Edge Stable 123.
 
 
 <!-- ====================================================================== -->
 ## Show issues in Elements
 <!-- checkbox has a (?) link to this anchor wording -->
 
-Enable this experiment to view syntax errors under HTML in the **DOM** view of the **Elements** tool. For more information, see [Wavy underlines highlight code issues and improvements in Elements tool](../whats-new/2021/04/devtools.md#wavy-underlines-highlight-code-issues-and-improvements-in-elements-tool).
+Displays syntax errors under HTML in the **DOM** view of the **Elements** tool.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:741]:
```suggestion
Displays syntax errors as wavy underlines under DOM nodes in the **Elements** tool.
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 18]
```diff
@@ -902,12 +912,54 @@ This is a programmatic way to record JavaScript execution, which can be useful i
...
+
 Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 125.
+* This checkbox is present in Microsoft Edge Stable 123.
+
+
+<!-- ====================================================================== -->
+## Timeline: Enable track confirmation feature that can reorder or hide a track in the flame chart
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:924]:
```suggestion
## Timeline: Enable track configuration feature that can reorder or hide a track in the flame chart
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 19]
```diff
@@ -902,12 +912,54 @@ This is a programmatic way to record JavaScript execution, which can be useful i
...
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 125_
 
+Enables the track confirmation feature, which can reorder or hide a track in the flame chart.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:929]:
```suggestion
Enables the track configuration feature, which lets you choose which tracks are visible in the **Performance** tool, and the order in which they appear. After you enable this experiment, to configure tracks:

1. Create a new performance recording in the **Performance** tool.
1. Hover over the tracks in the **Performance** tool, such as **Network**, **Interactions**, or **Main**. The track configuration icon (pen icon) appears on the left of the hovered track.
1. Click the track configuration icon. The track configuration screen appears.
1. On the track configuration screen, click the up or down arrows to change the order of the tracks, or click on the eye icons to toggle tracks.
1. To confirm the new track configuration, click the checkmark icon to the right of a track. The performance recording is displayed, with the new track configuration.
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 13 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:930]: See comment above.

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 14 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:951]: The upstream bug that introduced this experiment is also private. So I wasn't able to get much information about this.
My current understanding is this: users can install add-ons in the browser that add new tracks to the list of tracks displayed in the Performance tool. One such add-on could add a new track that displays a series of events over time, just like the Main or Interactions tracks do, but would show other types of events.

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 19 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:929]: revised draft, will check against UI:
```
Enables the track configuration feature, which lets you choose which tracks are visible in the **Performance** tool, and the order in which they appear.  After you enable this experiment, to configure tracks:
1. In the **Performance** tool, create a new performance recording.
1. In the **Performance** tool, hover over a track, such as **Network**, **Interactions**, or **Main**.  The **Track configuration** (pen icon) button appears on the left of the hovered track.
1. Click the **Track configuration** (pen icon) button.  The **track configuration** screen opens.
1. On the **track configuration** screen, click the up or down arrows to change the order of the tracks, or click on the eye icons to toggle tracks.
1. To confirm the new track configuration, click the checkmark icon to the right of a track.  The performance recording is displayed, with the new track configuration.

See also:
* [Introduction to the Performance tool](../evaluate-performance/index.md)
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 18 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:924]: confirmed in UI (d/k why grayed out, i think we discussed or logged bug):
image

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 17 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:741]: replaced

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 15 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:656]: todo before Resolve: update What's New PR per this rewording.
```
Distinguishes events that are triggered by the `postMessage` method from other events that are displayed in the **Performance** tool, to easily investigate performance issues that are related to posting messages across various threads of an application.
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 14 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:951]: Trying to find out more about this experiment. Will report when I know more.

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 12 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:687]: Trying to find out more about this experiment. Will report when I know more.

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 16 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:658]: todo: before Resolve, update What's New PR:
```
This experiment improves the **Performance** tool's **Main** section to help you quickly identify `postMessage` events and handlers, by distinguishing `postMessage` events from other events.  Without this experiment, events that are triggered by dispatching and handling messages between threads of an application appear as generic scripting function-call events.  With this experiment enabled:
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 14 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:955]: Enables extensions that customize the appearance of user timing events in the Performance tool.
from @captainbrosset offline

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 12 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:691]: The "minimap-driven navigation" (aka "breadcrumbs", a feature that enables developers to crop into and focus on the more relevant parts of a trace, see WNIDT 122:
https://developer.chrome.com/blog/new-in-devtools-122?hl=en#perf-breadcrumbs)
and the "track data context-menu" (a feature that enables developers to merge and hide superfluous or irrelevant entries of a trace, cleaning up the flame charts and trees, see WNIDT 124:
https://developer.chrome.com/blog/new-in-devtools-124#hide-func)
This experiment added the ability to persist those inside trace files, and reapply them when importing traces that contain breadcrumbs and altered tracks.

from @captainbrosset offline

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 20]
```diff
@@ -641,52 +610,95 @@ See also:
...
+* This checkbox is not present in Microsoft Edge Stable 123.
+
+
+<!-- ====================================================================== -->
+## Enable save and load trace with annotations in Performance Panel
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 125_
+
+Enables save and load trace with annotations in the **Performance** tool.  Adds the ability to persist the following two items inside trace files, and reapply them when importing traces that contain breadcrumbs and altered tracks.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:690]:
```suggestion
Adds the ability to persist the following changes made to trace files in the **Performance** tool, and reapply them when importing traces:
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 21]
```diff
@@ -641,52 +610,95 @@ See also:
...
+
+* Minimap-driven navigation, also called "breadcrumbs".  This feature enables you to crop into and focus on the more relevant parts of a trace.  See [Breadcrumbs in the Timeline](https://developer.chrome.com/blog/new-in-devtools-122?hl=en#perf-breadcrumbs) in _What's new in DevTools (Chrome 122)_.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:692]:
```suggestion
* Changes made by using the breadcrumbs. This feature enables you to crop into and focus on the more relevant parts of a trace.  See [Breadcrumbs in the Timeline](https://developer.chrome.com/blog/new-in-devtools-122?hl=en#perf-breadcrumbs) in _What's new in DevTools (Chrome 122)_.
```

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md [hunk 22]
```diff
@@ -641,52 +610,95 @@ See also:
...
+
+* Track data context-menu.  This feature enables you to merge and hide superfluous or irrelevant entries of a trace, cleaning up the flame charts and trees.  See [Hide functions and their children in the flame chart](https://developer.chrome.com/blog/new-in-devtools-124#hide-func) in _What's new in DevTools (Chrome 124)_.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:694]:
```suggestion
* Changes made by using the track contextual menu.  This feature enables you to merge and hide superfluous or irrelevant entries of a trace, cleaning up the flame charts and trees.  See [Hide functions and their children in the flame chart](https://developer.chrome.com/blog/new-in-devtools-124#hide-func) in _What's new in DevTools (Chrome 124)_.
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md (hunk 21 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:692]:
```
* Changes made by using the breadcrumbs.  This feature enables you to crop into and focus on the more relevant parts of a trace.  See [Breadcrumbs in the Timeline](https://developer.chrome.com/blog/new-in-devtools-122#perf-breadcrumbs) in _What's new in DevTools (Chrome 122)_.
```
trimmed url

**Pull request merged**